samples/reflection/bfbs2json.c

```diff
@@ -76,14 +76,20 @@ void print_attributes(reflection_KeyValue_vec_t KV)
     printf("]");
 }
 
-void print_object(reflection_Object_table_t O)
+void print_object(reflection_Object_table_t Obj)
 {
     reflection_Field_vec_t Flds;
     reflection_Field_table_t F;
     size_t i;
+    const char *name;
 
-    Flds = reflection_Object_fields(O);
-    printf("{\"name\":\"%s\"", reflection_Object_name(O));
+    Flds = reflection_Object_fields(Obj);
+
+    name = reflection_Object_name(Obj);
+    if (!name)
+        name = "NULL";
+
+    printf("{\"name\":\"%s\"", name);
     printf(",\"fields\":[");
```

Comment:
This check should be reverted back to the original form - both because the change is, or should be, unnecessary, and because we need to understand what is going on if there really is a problem and not just a mistyped letter.